Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/CommonComponents/AdditionalFieldsKeyModal.tsx

```diff
@@ -74,6 +75,12 @@ function AdditionalFieldsKeyModal(props: IProps): JSX.Element {
         return handle;
     };
   
+    React.useEffect(() => {
+        if (props.Show && dataStatus === 'error') {
+            getData(0, 10, [], '', true);
+        }
+    }, [props.Show])
```

Comment:
It was meant to reset the dataStatus as you opened the modal -- because if you opened a bad table then tried to open a good table, the status was still 'error' and the error popup was shown, but I changed it to just update the dataStatus to 'unintiated' directly. Calling getData was an extra step